composer.json

```diff
@@ -35,7 +35,7 @@
     "require-dev": {
         "johnkary/phpunit-speedtrap": "^1.0.1",
         "justinrainbow/json-schema": "^5.0",
-        "phpunit/phpunit": "^4.8.35 || ^5.4.3",
+        "phpunit/phpunit": "^5.4.3",
```

Comment:
we need to update schema ref in phpunit xml file

Reply:
Hehe, not anymore, see #3013!